src/qtoolkit/io/sge.py

```diff
@@ -0,0 +1,526 @@
+from __future__ import annotations
+
+import re
+import xml.dom.minidom
+import xml.parsers.expat
+from datetime import timedelta
+
+from qtoolkit.core.data_objects import (
+    CancelResult,
+    CancelStatus,
+    ProcessPlacement,
+    QJob,
+    QJobInfo,
+    QResources,
+    QState,
+    QSubState,
+    SubmissionResult,
+    SubmissionStatus,
+)
+from qtoolkit.core.exceptions import OutputParsingError, UnsupportedResourcesError
+from qtoolkit.io.base import BaseSchedulerIO
+
+# 'http://www.loni.ucla.edu/twiki/bin/view/Infrastructure/GridComputing?skin=plain':
```

Comment:
This link seems not reachable

Reply:
I replaced it now with other documentation links. I took this link from the aiida sge schedulers